src/helpers/api/retry.ts

```diff
@@ -0,0 +1,11 @@
+export function retry(fn: () => any, retries = 3, error = '') {
```

Comment:
Why not arrow function? Also, for more than three arguments, this could become an object instead. 👍